# Progress

# Introduction

## Why clustering scHi-C data?

Understanding the 3D genome organization:

- cellular diversity: individual cells can have different chromatin folding patterns.
- functional implications: spatial arrangement of the genome effects gene regulation, expression, and ultimately cell function.
- disease insights: abnormal chromatin organization is linked to deseases like cancer; clustering can help identify such aberrations.

Revealing hidden cellular states:

- identifying subpopulations: uncover subgroups of cell with distinct genome architectures.
- developmental processes: during differentiation, chromatin structure changes; clustering can map these transitions. [^1]

keywords: GWA, Hi-C, scHi-C, clustering

# Motivation

To cluster single-cell Hi-C data, often require heavy works on feature extraction and imputation.
And clustering is just a downstream task performed on the embeddings generated by these methods.

There still are previous approaches like k-nn Minhash and LDA are more direct in their clustering focus.

Advancing computational methods:

- challenges drive innovation: the unique properties of scHi-C data push for the development of novel computational approaches.

# Challenges for analysis based on scHi-C data

- Intrinsic variability
- Data sparsity
- Coverage heterogeneity
- High dimension

# the main pipeline

- How to cluster scHi-C data?
- imputation
- feature extraction (dimension reduction)
- clustering

# Clustering Method

## Clustering Framework

- scHiCluster
- scHiC-topic
- Higashi
- Muscle
- SnapATAC2
- (tensor decomposition)

## Previous Clustering Method

- K-Means, Hierarchical Clustering, DBSCAN

## My Clustering Method

# (Interpretability?)

# My Result

# Conclusion

# Contribution of this work

# References

[^1]: Linking genome structures to functions by simultaneous single-cell Hi-C and RNA-seq
